support/locust/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:latest
 WORKDIR /locust
 EXPOSE 8089
 COPY . /locust/src
-COPY sample_code /locust/sample_code
+# COPY sample_code /locust/sample_code
```

Comment:
Is this not needed in the docker?

Reply:
It is needed in Docker. But it is not needed in image. 
COPY command loads it into the image.

Reply:
I have pushed a new commit and deleted that useless comment line.